cli/utils/server.js

```diff
@@ -17,7 +17,7 @@ let server;
  * Starts the httpServer
  * @name start
  */
-function start(root, distPath) {
+function start(argv, root, distPath) {
```

Comment:
Just confirming that nothing outside of Builder calls this function (e.g., `skyux-sdk-builder-config`, `sdk-pact`, etc.)? If so, putting `argv` at the beginning of the argument list would be a breaking change.

Reply:
Nothing to my knowledge should be calling this method.  I could move it to the end to avoid that I suppose.  I like it in the front as that follows our current pattern.  If something outside of builder is calling it, they'd have to make a change to pass it in anyway.  I could go either way, but am leaning towards proceeding with the understanding it is not publicly exposed.

Reply:
Sounds good.